variables.tf

```diff
@@ -502,3 +502,21 @@ variable "use_legacy_network_interface" {
   nullable    = false
   default     = false
 }
+
+##############################################################################
+# Dedicated Host Variables
+##############################################################################
+
+variable "enable_dedicated_host" {
```

Comment:
add validation to check if a value is passed for `dedicated_host_id` when `enable_dedicated_host` is set to `true`.

Reply:
Updated